### PR TITLE
RAINCATCH-1254 - Review WFM introduction

### PR DESCRIPTION
**Pull request by witmicko**
## Motivation:
Review Raincatcher introduction

## JIRA
https://issues.jboss.org/browse/RAINCATCH-1254

@wtrocki @austincunningham please have a look at this.

### CODE REVIEW

**Comment by wtrocki** [comment]
@witmicko - Great work. Let's merge this PR but please do not close ticket - you may reassign that to myself.

I have a lots of contents including diagrams that will need to be included here. We looking into 2-3 pages of content. Most of the content were sent already by different parties etc. Content we have at the moment it's not covering important aspects. Really sorry, but actually introduction it's really important topic and key to the success of our project. 


screen shot 2017-09-09 at 6 51 07 pm

**Review by wtrocki — DISMISSED** [review]

**Comment by wtrocki** [comment]
@jstaffor - diagrams will be only in upstream and as support material.

File: docs/workforce-management-framework/topics/con-introducing-raincatcher.adoc [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
+{WFM-RC-NameShort} allows developers to leverage existing demo-solution by extending it to suit their business needs.
 
-The following features are on the {WFM-RC-NameShort} roadmap:
 
+Main features in the latest {WFM-RC-Version}:
 * Workflow Management
-* Geolocation
+* Real-time Data Access
+* Data Synchronization
+
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/con-introducing-raincatcher.adoc:19]: * Pluggable authentication

File: docs/workforce-management-framework/topics/con-introducing-raincatcher.adoc [hunk 2]
```diff
@@ -1,23 +1,28 @@
...
 
-The primary goal of {WFM-RC-NameShort} is to produce a set of modules for developers to integrate {WFM-RC-NameShort} features into their applications.
+The primary goal of {WFM-RC-NameShort} is to ease mobile enablement efforts.
+It achieves that by providing set of modules that can be integrated into existing applications additionally
+{WFM-RC-NameShort} allows developers to leverage existing demo-solution by extending it to suit their business needs.
 
-The following features are on the {WFM-RC-NameShort} roadmap:
 
+Main features in the latest {WFM-RC-Version}:
 * Workflow Management
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/con-introducing-raincatcher.adoc:16]: * Workflow Management
** Editable workflow
** Assignable workorders
** Customizable steps

File: docs/workforce-management-framework/topics/con-introducing-raincatcher.adoc [hunk 3]
```diff
@@ -1,23 +1,28 @@
 [id='con-introducing-raincatcher-{chapter}']
 = Introducing {WFM-RC-NameLong}
-
 //downstream == {WFM-RC-NameLong} is a Workforce Management Platform and is also referred to as {WFM-RC-NameShort}.
+
 {WFM-RC-NameLong} is a Workforce Management Platform.
-The latest version of {WFM-RC-NameShort} is {WFM-RC-Version}.
```
Comment (jstaffor) [inline docs/workforce-management-framework/topics/con-introducing-raincatcher.adoc:6]: {WFM-RC-NameShort} connects a business’ back-office with its fleet of mobilized employees.
{WFM-RC-NameShort} leverages commoditized mobile hardware, making use of each employee’s mobile phone.

**Review by jstaffor — DISMISSED** [review]

**Comment by wtrocki** [comment]
@jstaffor - this introduction is in really bad shape. We missing some general information and content is not clear. I'm going to merge that into the master, as content is there already and review and rebuild this chapter from scratch. I have applied your suggestions and merging it back.

**Review by wtrocki — APPROVED** [review]

**Pull request merged**